packages/@aws-cdk/aws-ecs/lib/base/task-definition.ts

```diff
@@ -265,8 +290,8 @@ export class TaskDefinition extends TaskDefinitionBase {
       throw new Error('Cannot set placement constraints on tasks that run on Fargate');
     }
 
-    if (this.isFargateCompatible && (!props.cpu || !props.memoryMiB)) {
-      throw new Error(`Fargate-compatible tasks require both CPU (${props.cpu}) and memory (${props.memoryMiB}) specifications`);
+    if (this.isFargateCompatible && (!this.cpu || !this.memoryMiB)) {
```

Comment:
I wouldn't necessarily have changed this. Now it doesn't matter, strictly speaking, but what's the advantage?

Reply:
It was more for the sake of verifying that both of those properties were defined, that being said, I'll change it back.